src/wp-includes/default-constants.php

```diff
@@ -13,7 +13,7 @@
  * @since WP-3.0.0
  *
  * @global int    $blog_id    The current site ID.
- * @global string $wp_version The ClassicPress version string.
+ * @global string $wp_version The WordPress version string.
```

Comment:
This should ultimately be ClassicPress, but this change should be tackled separately, so leaving it as WordPress is correct for now.  See https://github.com/ClassicPress/ClassicPress/issues/58.